src/components/icon.js

```diff
@@ -14,6 +14,9 @@ export const graphics = {
               <path d="m21.756592,0l-19.707765,0c-1.129684,0 -2.048827,0.919143 -2.048827,2.048827l0,19.707765c0,1.129683 0.919143,2.048828 2.048827,2.048828l19.707765,0c1.129683,0 2.048828,-0.919146 2.048828,-2.048828l0,-19.707765c0,-1.129684 -0.919146,-2.048827 -2.048828,-2.048827zm-19.707765,1.756137l19.707765,0c0.161371,0 0.29269,0.13132 0.29269,0.292689l0,2.880651l-20.293145,0l0,-2.880651c0,-0.161369 0.13132,-0.292689 0.292689,-0.292689zm19.707765,20.293145l-19.707765,0c-0.161369,0 -0.292689,-0.131319 -0.292689,-0.29269l0,-15.070976l20.293145,0l0,15.070976c0,0.161371 -0.131319,0.29269 -0.29269,0.29269z" id="svg_2"/>
               <path d="m15.561133,13.561283l-2.780355,0l0,-2.780355c0,-0.484987 -0.393082,-0.878069 -0.878068,-0.878069s-0.878069,0.393082 -0.878069,0.878069l0,2.780355l-2.780356,0c-0.484987,0 -0.878068,0.393084 -0.878068,0.878069s0.393082,0.878069 0.878068,0.878069l2.780356,0l0,2.780355c0,0.484985 0.393081,0.878069 0.878069,0.878069s0.878068,-0.393084 0.878068,-0.878069l0,-2.780355l2.780355,0c0.484986,0 0.87807,-0.393083 0.87807,-0.878069s-0.393084,-0.878069 -0.87807,-0.878069z" id="svg_3"/>
             </g>),
+  arrow: (<g id="chevron-right">
+            <polygon points="94.35,0 58.65,35.7 175.95,153 58.65,270.3 94.35,306 247.35,153"/>
```

Comment:
Ah, haha, I totally forgot we even had this file. Yeah, let's stick our logo in here too and use this arrow :-)